modules/afsocket/transport-mapper-inet.c

```diff
@@ -144,17 +143,27 @@ transport_mapper_inet_construct_log_transport(TransportMapper *s, gint fd)
 {
   TransportMapperInet *self = (TransportMapperInet *) s;
 
+  gboolean is_multi = FALSE;
+  LogTransport *transport = NULL;
+
   if (self->tls_context && _is_tls_required(self))
     {
-      return _construct_tls_transport(self, fd);
+      transport = _construct_tls_transport(self, fd);
     }
-
-  if (self->tls_context)
+  else if (self->tls_context)
+    {
+      is_multi = TRUE;
```

Comment:
This is not true, when `self->super.create_multitransport` everything will be created as a multi_transport

Reply:
the variable name here is misleading, it should refer to if the transport must be switched in the proxy
i will rename it